site/homepage/content/release-notes.md

```diff
@@ -25,6 +25,11 @@ title = "Release Notes"
   mechanisms advertised to a client connecting to these components. This can be used to
   restrict the support to only one of the SASL PLAIN and EXTERNAL mechanisms instead of both. 
 
+### API Changes
+
+ * The device registry credentials endpoint will no longer handout details for hashed-password secrets.
```

Comment:
please be more explicit about which information is no longer returned

Reply:
I updated. Let me know what you think.